### PR TITLE
Fix AdminNetworkPolicy yamls

### PR DESCRIPTION
**Pull request by tssurya**
Version(s):

Issue:

Link to docs preview:

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by JoeAldinger — APPROVED** [review]
/lgtm

File: modules/nw-ovn-k-adminnetwork-policy-action-rules.adoc [hunk 1]
```diff
@@ -31,9 +31,8 @@ spec:
     action: "Allow"
     from:
     - namespaces:
-        namespaceSelector:
-          matchLabels:
-            kubernetes.io/metadata.name: monitoring
+        matchLabels:
+          kubernetes.io/metadata.name: monitoring
 # ...
```
Comment (asood-rh) [inline modules/nw-ovn-k-adminnetwork-policy-action-rules.adoc:36]: I believe these indicate this is snippet of policy not complete policy.
@JoeAldinger

Comment (JoeAldinger) [reply modules/nw-ovn-k-adminnetwork-policy-action-rules.adoc:36]: Yes

**Pull request merged**